Tools/ML/model.h

```diff
@@ -20,23 +20,23 @@
 #ifndef TOOLS_ML_MODEL_H_
 #define TOOLS_ML_MODEL_H_
 
-// C++ and system includes
+#include <onnxruntime_c_api.h>
```

Comment:
This is wrong, no?

Reply:
It provides `OrtAllocatorType` and `OrtMemType`.

Reply:
In the `else` branch?